### ISSUE TITLE
use '"' and !r in f-string detected Unexpected token at end of expression

### ISSUE THREAD

**Issue by nanarino** [microsoft/pylance-release#1584]
## Environment data

- Language Server version: v2021.7.22
- OS and version: Win10
- Python version (& distribution if applicable, e.g. Anaconda): 3.9

## Expected behaviour
```python
s = "#M's#"
print(f"""{s.replace("'",'')}""") # ok
print(f"""{s.replace("'",'')!r}""") # Unexpected token at end of expression
import datetime
print(f"""{datetime.datetime.now().isoformat(sep="'")}""")  # ok
print(f"""{datetime.datetime.now().isoformat(sep="'")!r}""") # Unexpected token at end of expression
```

## Actual behaviour

I actually use it to splice sqlite3 comman，the quotation marks need to be replaced
Normal output and sqlite3 used ok

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next release.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2021.7.5, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202175-22-july-2021
